app/helpers/application_helper.rb

```diff
@@ -96,7 +96,7 @@ def facet_desc_sort!(items=[])
   def fixed_work_type_order(items=[])
     model_list = []
     items.each {|item| model_list.push(item) }
-    fixed_ordered_list = [Article, EescPublication, GraduateThesisOrDissertation, TechnicalReport, GraduateProject, AdministrativeReportOrPublication, HonorsCollegeThesis, ConferenceProceedingsOrJournal, UndergraduateThesisOrProject, OpenEducationalResource, Dataset, Default]
+    fixed_ordered_list = [Article, EescPublication, GraduateThesisOrDissertation, TechnicalReport, GraduateProject, AdministrativeReportOrPublication, HonorsCollegeThesis, ConferenceProceedingsOrJournal, UndergraduateThesisOrProject, OpenEducationalResource, Dataset, Default, nil, PurchasedEResource]
```

Comment:
what purpose is the `nil` here for?

Reply:
Steve specifically wanted it to show up under default in the last row. So we had to skip a cell in the modal view.

Reply:
<img width="910" alt="screen shot 2018-02-16 at 3 38 07 pm" src="https://user-images.githubusercontent.com/6424683/36334069-7e9830e6-132f-11e8-8497-373cb4c960f1.png">

Reply:
```
vantuyls [2:11 PM]
bottom of the right side
keep two columns
if this creates a blank space at bottom left, thats fine
```